src/lib/model/ticket/input-field-request.ts

```diff
@@ -0,0 +1,26 @@
+import { UUID } from '../uuid';
+
+/**
+ * Set an input field of a ticket.
+ *
+ * One of 2 options are allowed:
+ * (inputFieldId, value) or (inputFieldId, optionIds)
+ */
+export type InputFieldRequest =
+  | ValueInputFieldRequest
+  | OptionIdsInputFieldRequest;
+
+export interface BaseInputFieldRequest {
```

Comment:
BaseInputFieldRequest probably should not be exported? It's not exposed in the public api and not used anywhere else

Reply:
Fixed it!